core/src/i18n/en.yaml

```diff
@@ -188,6 +188,7 @@ catalog:
   subtitle: Enrich your experience with additional services that are installed outside the cluster.
   title: Service Catalog
 certificates:
+  annotations: Annotations
```

Comment:
We can use `common.headers.annotations` instead, removing this value. I do know that annotations are duplicated all over the place anyway (`role-bindings.labels.annotations`, `secrets.labels.annotations`)...